elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -172,10 +188,17 @@ def parse(element_html, data):
         a_sub = a_sub.strip()
 
         # Convert safely to sympy
-        a_sub_parsed = phs.convert_string_to_sympy(a_sub, variables)
+        a_sub_parsed = phs.convert_string_to_sympy(a_sub, variables, allow_complex=allow_complex)
 
-        # Store result as a string.
-        a_sub_string = str(a_sub_parsed)
+        # If complex numbers are not allowed, raise error if expression has the imaginary unit
+        if (not allow_complex) and (a_sub_parsed.has(sympy.I)):
+            s = 'Your answer was simplified to this, which contains a complex number (denoted $i$): $${:s}$$'.format(sympy.latex(a_sub_parsed))
```

Comment:
If the student inputs a $j$, will it be rendered as $i$ when displayed back to the student? The ECE folks would rather retain $j$ as $j$, I guess.

Reply:
done